.env

```diff
@@ -1,4 +1,4 @@
-DB_HOSTNAME=localhost
+DB_HOSTNAME=lutedb
```

Comment:
This should stay as localhost for the DB_HOSTNAME.  Perhaps the docker compose environment can set that, because DB_HOSTNAME = lutedb is really only applicable for docker.